src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -5790,7 +5790,7 @@ static object F(string s !!)
             var expectedCode =
 @"class C
 {
-    static object F(string s!!)
+    static object F(string s !!)
```

Comment:
Does it make sense to keep this test at all?

Reply:
No, let's delete it.